### DISCUSSION TITLE
Getting rid of a Hacker

### DISCUSSION THREAD

**Discussion by HackThis8822** [community/community#50706] [Pull Requests]
Hey guys, 

For like four months now my iPhone has been remote controlled remotely day and night by and unknown source. Webpages are changed, passwords are changed, things are open and closed in front of me.

#Any suggestions on how to get rid of this person?? 

I've tried everything from getting a VPN, to finding and deactivating a secret Nord VPN and Cloudflasre account they've had.

**Comment by Lugozambia** [comment]
How can get free internet

**Comment by Lugozambia** [comment]
How can I make free internet
